includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -541,6 +561,25 @@ public function display_test_mode_notice() {
 		}
 	}
 
+	/**
+	 * Add notice explaining that the selected currency is not available.
+	 */
+	public function display_not_supported_currency_notice() {
```

Comment:
If we change this to be displayed in the admin in general, we need to make sure to have a check here to make sure the user should be seeing the notice. An example of usage is here: https://github.com/Automattic/woocommerce-payments/blob/develop/includes/multi-currency/AdminNotices.php#L35-L37

```
		if ( ! current_user_can( 'manage_woocommerce' ) ) {
			return;
		}
```

Reply:
Also added in https://github.com/Automattic/woocommerce-payments/pull/2558/commits/36c42a038a07a152c29bb1706da2731d18d72e3e